src/Core/src/Platform/Android/EditTextExtensions.cs

```diff
@@ -200,6 +200,61 @@ public static void UpdateReturnType(this AppCompatEditText editText, IEntry entr
 			editText.ImeOptions = entry.ReturnType.ToNative();
 		}
 
+		[PortHandler]
+		public static void UpdateCursorPosition(this AppCompatEditText editText, IEntry entry)
+			=> UpdateCursorSelection(editText, entry);
+
+		[PortHandler]
+		public static void UpdateSelectionLength(this AppCompatEditText editText, IEntry entry)
+			=> UpdateCursorSelection(editText, entry);
+		
+
+		static void UpdateCursorSelection(AppCompatEditText editText, IEntry entry)
```

Comment:
Is the body correct? Is it related to CursorPosition and not SelectionLength?

```suggestion
		static void UpdateCursorPosition(AppCompatEditText editText, IEntry entry)
```

Reply:
CursorPosition and SelectionLength use the same body because they are essentially dependent on one another. I tried to de-couple them but was unable to achieve the same exact functionality as in Xamarin.Forms. The method body is related to both *CursorPosition* and *SelectionLength*. Here are the references I used:
- Xamarin.Forms.Android CursorPosition & SelectionLength calling *UpdateCursorSelection* <a href="https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs#L284">here</a>
- Xamarin.Forms.Android UpdateCursorSelection Implementation <a href="https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs#L443">here</a>

Does that help?

Reply:
Ahh yes, that makes sense. Thanks for clarifying!
@jsuarezruiz do you think it wold be helpful to add a comment here or to update the name to be UpdateCursorPositionAndSelectionLength (or something similar) for clarity?

Reply:
The same thing goes for iOS as well so I think this comment applies there as well.

Reply:
Yes, let's include a comment in the different platforms like Rachel suggested.